Create ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md (#1909)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-18-1-fix-data-point-list-widget-redirecting-assets-devices-incorrectly-5831-graft.md

---
date: ""

> title: Data point list widget now redirects to the correct asset's devices

Reviewer: ```suggestion
title: Data point list widget now redirects to the correct assets
```

> title: Clicking on an asset name in the "Data point list" widget now redirects to the correct asset page.

Reviewer 2: ```suggestion
title: Asset links in the "Data point list" widget now redirect to the correct URL
```

> title: fix: Data point list widget redirecting assets devices incorrectly (#5831) [GRAFT][release/cd] (#6060)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Data point list widget now redirects to the correct asset's devices
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57677
version: 1019.18.1
---

> fix: Data point list widget redirecting assets devices incorrectly (#5831) [GRAFT][release/cd] (#6060)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, when clicking on a data point in the data point list widget, users were sometimes redirected to the wrong asset's devices. This issue has now been resolved. With this fix, clicking on a data point in the data point list widget will always take the user to the correct asset's devices as expected. The change improves the usability and navigation within the application.
```

> In the past, when clicking on a data point in the data point list widget, users were sometimes redirected to the wrong asset's devices. This issue has now been resolved. With this fix, clicking on a data point in the data point list widget will always take the user to the correct asset's devices as expected. The change improves the usability and navigation within the application.

Reviewer: ```suggestion
In the past, when clicking on the asset name in the data point list widget, users were sometimes redirected to the wrong url. This issue has now been resolved. With this fix, clicking on an asset name in the data point list widget will now redirect the user to the correct asset page.
```
